### ISSUE TITLE
Setting border color also sets key/command help color

### ISSUE THREAD

**Issue by tesu** [ranger/ranger#891]
#### ISSUE TYPE
- Improvement/feature request

#### RUNTIME ENVIRONMENT

- Operating system and version: Arch Linux
- Terminal emulator and version: urxvt v9.22
- Python version: 3.6.1
- Ranger version/commit: 1.8.1
- Locale: en_US

#### EXPECTED BEHAVIOR
`context.border` should only be true for the borders, instead of including the key/command hint text.

#### CURRENT BEHAVIOR
`context.border` returns true for both the borders and the key/command hint text, so they cannot be differentiated between in the colorscheme.

#### CONTEXT
I was having issues with flickering image previews, and turning on `draw_borders` fixed it. However, I didn't really like the look of the borders, so I wanted to just set the borders to black so that I could have image preview work while not actually seeing the borders.

I threw together a quick color scheme to do this:
```
from __future__ import (absolute_import, division, print_function)

from ranger.colorschemes.default import Default
from ranger.gui.color import black, blue

class Scheme(Default):
    progress_bar_color = blue

    def use(self, context):
        fg, bg, attr = Default.use(self, context)

        if context.border:
            return black, bg, attr

        return fg, bg, attr
```
The border was now gone for me, but now the hint text is also unreadable. I've included a picture below showing how it looks after I press g.
#### POSSIBLE SOLUTIONS
Have a separate field for the hint text so that it can be differentiated from the border in the colorscheme, so the two can be styled differently.

#### IMAGES / VIDEOS
![image](https://user-images.githubusercontent.com/840752/27412630-e33fd126-56c3-11e7-83f0-ec3750f3ad31.png)

**Comment by tesu** [comment]
Curiously enough, turning `draw_borders` off makes it so that `context.border` no longer styles the key/command help text.

**Comment by hut** [comment]
Fixed in #893, thank you
